ocs_ci/utility/utils.py

```diff
@@ -1373,7 +1373,11 @@ def email_reports(session):
     # total = passed + failed + error
     # percentage_passed = (passed / total) * 100
 
-    build_id = get_ocs_build_number()
+    try:
+        build_id = get_ocs_build_number()
+    except Exception as e:
+        build_id = ""
+        log.exception(e)
```

Comment:
I believe this will result in an empty log message followed by the stack trace. `log.exception` will automatically grab the stack trace for us so we don't need to pass `e` to it. We should instead include a more informative message about what we were doing when we hit that error.

```python
try:
    assert True == False
except AssertionError as e:
    log.exception(e)
```
```
ERROR:__main__:
Traceback (most recent call last):
  File "<ipython-input-9-28d4662577e9>", line 2, in <module>
    testing()
  File "<ipython-input-3-899b3bb10609>", line 2, in testing
    assert True == False
AssertionError
```
```python
try:
    assert True == False
except AssertionError:
    log.exception('some specific message about failing to get build number')
```
```
ERROR:__main__:some specific message about failing to get build number
Traceback (most recent call last):
  File "<ipython-input-10-c8c132c2e320>", line 2, in <module>
    testing()
  File "<ipython-input-3-899b3bb10609>", line 2, in testing
    assert True == False
AssertionError
```

Reply:
Thanks for the notice, I copied it from the following lines - will fix it there also.

Reply:
Fixed in 50315f6.